reports/enwiki/emptycats.py

```diff
@@ -61,7 +61,8 @@ def get_table_rows(self, conn):
                         AND tl_namespace = 10
                         AND (tl_title = 'Empty_category' OR
                              tl_title = 'Possibly_empty_category' OR
-                             tl_title = 'Monthly_clean-up_category'));
+                             tl_title = 'Monthly_clean-up_category' OR
+                             tl_title = 'SpeedyMaintanceCat'));
```

Comment:
`SpeedyMaintanceCat` is a template redirect that contains a misspelling. It would be better to use the target template name instead: `Maintenance_category_autotag`.

Reply:
Done